### PR TITLE
Rails performance and Honeybadger

### PR DESCRIPTION
**Pull request by hdamico**
## What does this PR do?

Implements Rails Performance for performance monitoring and HoneyBadger for exceptions monitoring.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -50,6 +50,10 @@ gem 'chartkick', '~> 3.3', '>= 3.3.1'
 
 gem 'sidekiq-cron', '~> 1.1'
 
+gem 'rails_performance', '~> 0.9.1'
```
Comment (hvilloria) [inline Gemfile:53]: comment to test the github webhook

Comment (santib) [inline Gemfile:53]: can we also add rack mini profiler? I'd really like to learn about that tool 😬 https://github.com/MiniProfiler/rack-mini-profiler

File: config/initializers/rails_performance.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+if defined?(RailsPerformance)
+  RailsPerformance.setup do |config|
+    config.redis    = Redis.new(url: ENV.fetch('REDIS_PERFORMANCE_URL') { 'redis://localhost:6379/1' })
+    config.duration = 4.hours
+    config.enabled  = true
+    config.http_basic_authentication_enabled = false
```
Comment (santib) [inline config/initializers/rails_performance.rb:6]: don't we need to have this enabled so that only admins access the performance panel?

Comment (hdamico) [reply config/initializers/rails_performance.rb:6]: Sure, do i hardcode the credentials aswell as credentials for activeadmin? Or do you preffer to save them in the ENV and use the same for both?

File: Gemfile (hunk 1 above)
Comment (hdamico) [reply Gemfile:53]: Sure!

File: config/initializers/rails_performance.rb (hunk 2 above)
Comment (santib) [reply config/initializers/rails_performance.rb:6]: maybe env variables is better, or if you can authorize by `AdminUser` then better

Comment (santib) [reply config/initializers/rails_performance.rb:6]: not sure if it's possible with this gem

File: Gemfile [hunk 3]
```diff
@@ -50,6 +50,10 @@ gem 'chartkick', '~> 3.3', '>= 3.3.1'
...
+
+gem 'honeybadger', '~> 4.0'
```
Comment (santib) [inline Gemfile:55]: should we also add exception hunter? or not yet? cc @brunvez

File: config/initializers/rails_performance.rb (hunk 2 above)
Comment (hdamico) [reply config/initializers/rails_performance.rb:6]: Ok, I will check, otherwise I use ENV

File: Gemfile (hunk 3 above)
Comment (brunvez) [reply Gemfile:55]: We still need to release it to RubyGems, I plan to have it by Monday. I don't want to be a blocker so maybe we can do a different PR.

**Pull request merged**